src/components/Spotlight/Spotlight.tsx

```diff
@@ -4,16 +4,29 @@ import { openSpotlight, SpotlightProvider } from "@mantine/spotlight";
 import { IconSearch } from "@tabler/icons";
 import { useNavigate } from "react-router-dom";
 import { RouteProperties, ROUTES } from "../RouteWrapper";
+import React, { useEffect, useRef } from "react";
```

Comment:
Looks like you did not end up using these so you can probably remove. 
```suggestion

```

Reply:
Ah yep, removing.